README.md

```diff
@@ -1,116 +1,209 @@
-[![Snyk logo](https://snyk.io/style/asset/logo/snyk-print.svg)](https://snyk.io)
+# Snyk Security
 
-[![Latest Release](https://img.shields.io/github/v/release/jenkinsci/snyk-security-scanner-plugin)](https://github.com/jenkinsci/snyk-security-scanner-plugin/releases)
+[![Homepage](https://img.shields.io/jenkins/plugin/v/snyk-security-scanner.svg)](https://plugins.jenkins.io/snyk-security-scanner)
+[![Changelog](https://img.shields.io/github/release/jenkinsci/snyk-security-scanner-plugin.svg?label=changelog)](https://github.com/jenkinsci/snyk-security-scanner-plugin/releases)
+[![Installs](https://img.shields.io/jenkins/plugin/i/snyk-security-scanner.svg)](https://plugins.jenkins.io/snyk-security-scanner)
+[![Vulnerabilities](https://snyk.io/test/github/jenkinsci/snyk-security-scanner-plugin/badge.svg)](https://snyk.io/test/github/jenkinsci/snyk-security-scanner-plugin)
 
-[![Known Vulnerabilities](https://snyk.io/test/github/jenkinsci/snyk-security-scanner-plugin/badge.svg)](https://snyk.io/test/github/jenkinsci/snyk-security-scanner-plugin)
+[![Snyk](https://snyk.io/style/asset/logo/snyk-print.svg)](https://snyk.io)
 
-***
+Test and monitor your projects for vulnerabilities with Jenkins. Officially maintained by [Snyk](https://snyk.io).
 
-# Table of Contents
-- [Snyk Security Scanner](#snyk-security-scanner)
-- [Configuration](#configuration)
-  - [Global Configuration](#global-configuration)
-  - [Project Configuration](#project-configuration)
-    - [Freestyle Jobs](#freestyle-jobs)
-    - [Pipeline Jobs](#pipeline-jobs)
-- [Release Workflow](#release-workflow)
-  - [Performing a Plugin Release](#performing-a-plugin-release)
-  - [Experimental Plugin Releases](#experimental-plugin-releases)
+## Usage
 
+To use the plugin up you will need to take the following steps in order:
 
-# Snyk Security Scanner
+1. Install the Snyk Security Scanner Plugin.
```

Comment:
Should these be links that'd direct to the right healine?

Reply:
Fixed